style.css

```diff
@@ -0,0 +1,41 @@
+/* remove default margin and padding from body */
```

Comment:
since this stylesheet is specific only to the map component, can we rename it to be more specific? something like mapStyles.css?